src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -168,24 +84,29 @@ def __str__(self) -> str:
 
 @dataclass(frozen=True)
 class DeadlineWorkerConfiguration:
-    operating_system: OperatingSystem
     farm_id: str
     fleet: Fleet
     region: str
-    user: str
-    group: str
     allow_shutdown: bool
     worker_agent_install: PipInstall
-    job_users: list[PosixSessionUser] = field(
-        default_factory=lambda: [PosixSessionUser("jobuser", "jobuser")]
-    )
     start_service: bool = False
     no_install_service: bool = False
     service_model_path: str | None = None
-    file_mappings: list[tuple[str, str]] | None = None
+
     """Mapping of files to copy from host environment to worker environment"""
-    pre_install_commands: list[str] | None = None
+    file_mappings: list[tuple[str, str]] | None = None
+
     """Commands to run before installing the Worker agent"""
+    pre_install_commands: list[str] | None = None
```

Comment:
I think it could be written either way. The worker config already contains a bunch of other implementation details of the worker, like the `worker_agent_install`. I'm hesitant to change it. If anything I'd remove it as 1) Any commands could be implemented as part of the `PipInstall` class (which really should be implementing an abstract class). 2) it's not being used anywhere, and 3) The current implementations of the workers actually run this AFTER the install commands, so the name is misleading anyways.